optimize_sidebars.js

```diff
@@ -945,4 +888,661 @@ module.exports = {
       ],
     },
   ],
+
```

Comment:
The rest of the changes in this file are to duplicate the entire sidebar structure of these two sections into this file. This duplication makes me super nervous because people will overlook it....but the sidebars change relatively infrequently, so hopefully it's not a constant problem.